src/frontpage/models/Offline.ts

```diff
@@ -6,5 +6,5 @@ export interface IOfflineIssue {
   issue: string;
   release_date: string;
   title: string;
-  image: IResponsiveImage;
+  image?: IResponsiveImage;
```

Comment:
I think this should be `IResponsiveImage | null` instead of possibly `undefined`. The API will never return undefined, as JSON does not support `undefined`, and all `None` values in Python are serialized to `null`. 

Very minor thing, but it might come back and bite us in the ass if we ever do a strict equality check instated of a a truthy/flasy check like above